server/api/scheduler.go

```diff
@@ -144,8 +144,10 @@ func (h *schedulerHandler) CreateScheduler(w http.ResponseWriter, r *http.Reques
 
 	case schedulers.GrantLeaderName:
 		h.addEvictOrGrant(w, input, schedulers.GrantLeaderName)
+		return
 	case schedulers.EvictLeaderName:
 		h.addEvictOrGrant(w, input, schedulers.EvictLeaderName)
```

Comment:
Maybe can modify the function `addEvictOrGrant` to return error, and then modify the function `CreateScheduler` to return the error in a uniform location?

Reply:
Yes, I have also considered this method, but I noticed that many different types of errors will be generated in the addEvictOrGrant method. However, there is no definition of the related "missing store id" error in the errs package.

Now I added the "missing store id" error type definition, return the error in addEvictOrGrant and handle it uniformly in the handleAddEvictOrGrantErr method

Reply:
If possible, could the function `CreateScheduler` be unified to return error as well? :)

Reply:
Do you mean not to use the handleAddEvictOrGrantErr method and return it directly in CreateScheduler?

Reply:
yes, can we handle error in `CreateScheduler`?

Reply:
If error handling is performed directly in CreateScheduler and returned, it will cause code redundancy in the GrantLeaderName branch and the EvictLeaderName branch.

Reply:
- maybe can delete function `addEvictOrGrant` to unify 2 branch and pass scheduler-name:
`case schedulers.GrantLeaderName, schedulers.EvictLeaderName:`

- what I want is to delete the redundant error handle
```
	var serverError error

	switch name {
	case schedulers.BalanceLeaderName:
		serverError = h.AddBalanceLeaderScheduler()		
	case schedulers.BalanceWitnessName:
		serverError = h.AddBalanceWitnessScheduler()
	...
	}
	if serverError != nil {
		...
	}
		
```